packages/server/lib/controllers/xhrs.js

```diff
@@ -69,10 +69,10 @@ module.exports = {
         .set(headers)
         .status(status)
         .end(chunk)
-      }).catch((err) => {
+      }).catch(({ message, stack, ...err }) => {
         return res
         .status(400)
-        .send({ __error: err.stack })
+        .send({ __error: { message, stack, ...err } })
```

Comment:
I have also another solution which doesn't require a changes here in the xhr controller:

```
diff --git a/packages/driver/src/cy/commands/xhr.js b/packages/driver/src/cy/commands/xhr.js
index 7b00ed70a..4075b8a04 100644
--- a/packages/driver/src/cy/commands/xhr.js
+++ b/packages/driver/src/cy/commands/xhr.js
@@ -200,7 +200,7 @@ const startXhrServer = (cy, state, config) => {
     },
 
     onFixtureError (xhr, err) {
-      err = $errUtils.cypressErr(err)
+      err = $errUtils.cypressErr({ message: err })
 
       return this.onError(xhr, err)
     },
```

But because `__error` which becomes `err` is a string of a stack trace, there is a lot of noise in error message :/ That's why I decided to pass message and stack separately. 

Let me know which solution is better.